unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/EventsSubSection/EventsSubSectionMenu/EventsSubSectionComponentView.cs

```diff
@@ -128,14 +128,23 @@ public interface IEventsSubSectionComponentView
     /// Set the current scroll view position to 1.
     /// </summary>
     void RestartScrollViewPosition();
+
+    /// <summary>
+    /// Configure the needed pools for the events instantiation.
+    /// </summary>
+    void ConfigurePools();
```

Comment:
Does it make sense for this to be in the interface? We could place it in the constructor/awake of the implementations and free the controllers from having to explicitly call this.

Reply:
In this case I am doing it intentionally, I'm ordering the creation of the pools before the view is activated because I want to have the pools prewarmed and ready before the user opens the Start Menu by first time.

Reply:
But you could do it in the Awake of each view and they would be prewarmed as well without the controller knowing.

Reply:
But the Awake is called when the monobehaviour is active, and in this case all the views are hidden until the user open the Start Menu by first time.

Reply:
Oh you are right!

Reply:
Anyway thanks for the suggestion, because it is applicable for other cases and I personally sometimes I use to pass too much responsibilities to the controller when they should be in the own view 😄